rust-connectors/common/src/config.rs

```diff
@@ -39,6 +39,9 @@ pub struct ConnectorConfig {
 
     #[serde(default, skip_serializing_if = "Option::is_none")]
     pub consumer: Option<ConsumerParameters>,
+
+    #[serde(default, skip_serializing_if = "Option::is_none")]
+    pub transform: Option<TransformParameters>,
```

Comment:
let's change this to `transforms` to be consistent

Reply:
Done